tests/smoke/fortran_mpi_driver.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+// other BLT Project Developers. See the top-level COPYRIGHT file for details
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+//----------------------------------------------------------------------
+
+/* 
+   fortran_mpi_test manages MPI_Init & MPI_Finalize. Do not put MPI
+   calls in this driver file -- put these calls in fortran_mpi_test,
+   which should be defined in blt_fruit_mpi_test.f90.
+ */
+extern "C" int fortran_mpi_test();
```

Comment:
[Likely outside the scope of this PR] Should the function / module be named `blt_fruit_mpi_test` for easier discoverability?
This would also imply changing `fortran_test` to `blt_fortran_test` in `fortran_driver.cpp` (and the related `blt_fruit_smoke.f90`.

Reply:
I think there are reasonable arguments for doing so in another PR, though obviously, this change would depend on whether the `fortran_driver.cpp` and `fortran_mpi_driver.cpp` files still exist.